profile/README.md

```diff
@@ -0,0 +1,58 @@
+## GraalVM on GitHub
+
+![GraalVM banner](https://raw.githubusercontent.com/graalvm/.github/main/graalvm-github.png "GraalVM on GitHub")
+
+GraalVM is a high-performance JDK designed to accelerate Java application performance while consuming fewer resources. 
+It provides the Graal compiler, which can be used as a just-in-time compiler to run Java applications on the HotSpot JVM or to ahead-of-time compile them into native executables. Besides Java, it also provides runtimes for JavaScript, Ruby, Python,  and a several other popular languages with polyglot capabilities.
+
+## Get Started
+
+* [Website](https://www.graalvm.org/)
+* [Documentation](https://www.graalvm.org/latest/docs/)
+* [Downloads](https://www.graalvm.org/downloads/)
+* [Docker images](https://github.com/orgs/graalvm/packages)
+* [Demos](https://github.com/graalvm/graalvm-demos)
+
+
+## GraalVM's GitHub Repositories
+
+Name         | Description
+------------ | -------------
+[Graal](https://github.com/oracle/graal) | The core repository
+[GraalVM CE Builds](https://github.com/graalvm/graalvm-ce-builds) | GraalVM CE builds
+[GraalVM CE Dev Builds](https://github.com/oracle/graal) | GraalVM CE Developer builds
+[Native Build Tools](https://github.com/graalvm/native-build-tools) | Build plugins for GraalVM Native Image
+[GraalVM Reachability Metadata](https://github.com/oracle/graalvm-reachability-metadata) | Reachability metadata for open-source libraries
+[Setup GraalVM](https://github.com/oracle/fastr) | GitHub Action for GraalVM
+[GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader) | GraalVM JDK Downloader
+[FastR](https://github.com/oracle/fastr) | Implementation of the R language
+[GraalJS](https://github.com/oracle/graaljs) | Implementation of JavaScript and Node.js
+[GraalPy](https://github.com/oracle/graalpython) | Implementation of the Python language
+[TruffleRuby](https://github.com/oracle/truffleruby) | Implementation of the Ruby language
+[GraalVM Demos](https://github.com/graalvm/graalvm-demos) | Several example applications illustrating GraalVM's capabilities
+[Simple Language](https://github.com/graalvm/simplelanguage) | A simple example language built with the Truffle framework
+[Simple Tool](https://github.com/graalvm/simpletool) | A simple example tool built with the Truffle framework
```

Comment:
I'd prefer the GitHub slug notation here, so instead of:
```
[Graal](https://github.com/oracle/graal) | The core repository
```
it'd be nice to have:
```
[`oracle/graal`](https://github.com/oracle/graal) | The GraalVM core repository
```

This will also avoid some weird duplication (e.g., `GraalVM CE Builds`).